.github/workflows/release.yml

```
@@ -379,7 +377,7 @@ jobs:
       # - run: sdkmanager --install "build-tools;30.0.3"
       - uses: malinskiy/action-android/emulator-run-cmd@release/0.1.7
         with:
-          cmd:  cd apps/flipper &&  dart run realm install &&  flutter test --dart-define=FLUTTER_TEST_ENV=true  integration_test/smoke_android_test.dart
+          cmd:  cd apps/flipper  &&  flutter test --dart-define=FLUTTER_TEST_ENV=true  integration_test/smoke_android_test.dart
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)